### ISSUE TITLE
Add support for Visual Studio 2022:(

### ISSUE THREAD

**Issue by israel-lpz** [madskristensen/JavaScriptPrettier#36]
### Description
It would be great if they added support for the new Visual Studio 2022

### Current behavior
It does not appear in the marketplace

### Expected behavior
Appear in the marketplace

**Comment by jack3898** [comment]
+1 to this, it would be really great. Shame it's not like VSCode where app versions aren't a concern at all, but I know they're different products.

**Comment by tommck** [comment]
@madskristensen I can take a crack at doing this for you

**Comment by tommck** [comment]
@madskristensen - actually, this is causing some difficulties.. missing project references, etc.  Seems like your project requires the Extensibility Tools package which isn't supported in VS2019 or VS2022 from what I can tell :(

**Comment by ahmedesoliman** [comment]
Yes, I upvote this. It would be great if you can support VS 2022 or support C# & C++ in the visual studio community.

**Comment by tommck** [comment]
@ahmedesoliman this can only support [what Prettier supports](https://prettier.io/docs/en/index.html)

**Comment by eklavya-raj** [comment]
I want Prettier its already 3rd quarter of 2022... still prettier is not working

**Comment by connormahern-MW** [comment]
Any updates on whether we will see Prettier Support for Visual Studio 2022 before the end of 2022???

**Comment by jesperbjensen** [comment]
@connormahern-MW I don't think this plugin is maintained anymore - and I don't think @madskristensen looks at the issues - or is simply too busy.

I ended up moving to Rider, because web development using modern techniques, that is not Blazor, is sub-par in Visual Studio (non code) - so I have not used this plugin since.

I think the better solutions is, for someone to fork it, and make an updated plugin - or just give up and use something better for web development, than Visual Studio - as I think that web developers using modern frontend toolchains, is not really the priority group for the full Visual Studio - as most of them is using Visual Studio Code.
The Visual Studio team is doing a great job - but they just have to prioritize - and MS already makes the most popular editor for web development. (vscode)

@madskristensen Is it an idea to make a note on the repository that this plugin has reached end-of-life? Maybe that will spark a main fork, that can add support for 2022 and beyond? If you have the time, then if you can give someone some notes about what have to be done to the plugin, to make it compatible with 2022 - then that would properly make it way easier for someone to take over.

**Comment by rwb196884** [comment]
Yeah, looks like this is dead.

What about https://marketplace.visualstudio.com/items?itemName=RichNewman.TypeScriptAnalyzerEslintPrettier

**Comment by gedri** [comment]
> What about https://marketplace.visualstudio.com/items?itemName=RichNewman.TypeScriptAnalyzerEslintPrettier

Anyone managed to use this to replace the Prettier plugin? 
How do I make it to simply format-on-save using the default prettier formatting rules and not introducing any other rules?
